dev_scripts/release-container-image.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python3
+"""
+Script to automate the release of a new Dangerzone container image.
+
+This script automates the manual steps described in the container image release
+process, including:
+1. Picking a release candidate image
+2. Attesting provenance and reproducibility
+3. Signing and publishing the image
+"""
+
+import json
+import re
+import subprocess
+import sys
+from pathlib import Path
+
+import click
+
+PROJECT_ROOT = Path(__file__).parent.parent
+IMAGE_NAME_PATH = PROJECT_ROOT / "share" / "image-name.txt"
+DOCKER_ENV_PATH = PROJECT_ROOT / "Dockerfile.env"
+
+
+def extract_debian_archive_date():
+    with DOCKER_ENV_PATH.open() as f:
+        content = f.read()
+
+    match = re.search(r"DEBIAN_ARCHIVE_DATE=(\d+)", content)
```

Comment:
I'm afraid we can't do that. If you recall, we bump the `DEBIAN_ARCHIVE_DATE` in our CI runners every night, so it's not necessarily the same as the stored one. So, we have to parse this  value from the image tag, as I mentioned in the doc.